analysis/include/PhysObject.hpp

```diff
@@ -120,8 +137,10 @@ class PhysObject {
   double d0;        ///< transverse impact parameter between track and beam spot
   double dxy, dz;   ///< distance between track and primary vertex
   double dxyErr, dzErr;///< uncertainties on dxy and dz
-  double vx, vy, vz;///< vertex coordinates
-  
+  int nVtx;
+  double  xVtx, yVtx, zVtx;///< vertex coordinates;Date:5/02/2023
+  double vx, vy, vz;
```

Comment:
What's the difference between vx and xVtx?

Reply:
I am not sure if they are different. I defined xVtx,.. for vertex info.